submissions/titilayoOlutola/HTML/Html task.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>FindMeABook</title>
+    </head>
+    <body>
+        <h1>FindMeABook</h1>
+
+        <h4> <em>your-library-on-the-go</em></h4>
+        <hr>
+        <tables>
+            <tr>
+                <td><a href="#">Fiction</a></td>
+                <td><a href="#">Non-Fiction</a></td>
+                <td><a href="#">Educational</a></td>
+                <td><a href="#">Autobiographies</a></td>
+                <td><a href="#">Historicals</a></td>
+                <td><a href="#">Others</a></td>
+            </tr>
+        </tables>
+    </body>
+    <section>
+        <h2>A Gentleman In Moscow By Amor Towles</h2>
+
+        <img src="Images/A_Gentleman_in_Moscow_(Amor_Towles).png" controls>
+        <img src="Images/Backview.jpeg" controls>
+        <h3><a href="#">DOWNLOAD</a></h3>
+        <h3><a href="#">ADD TO LIBRARY</a></h3>
+    </section>
+    <hr>
+    <section>
+        <h3>Description</h3>
+        <p><strong>File size: 10MB</strong></p>
+        <p><strong>
+            Pages: 462
+        </strong></p>
+        <p><strong>
+            Synopsis: With his breakout debut novel, Rules of Civility, Amor Towles established himself as a master of absorbing, sophisticated fiction, bringing late 1930s Manhattan to life with splendid atmosphere and a flawless command of style. Readers and critics were enchanted; as NPR commented, “Towles writes with grace and verve about the mores and manners of a society on the cusp of radical change.”
+
```

Comment:
Remove whitespace. Also line 39 IS too long. Use the enter key to break it into the next line.

Reply:
done